numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -1508,74 +1508,70 @@ PyUFunc_DefaultLegacyInnerLoopSelector(PyUFuncObject *ufunc,
     return raise_no_loop_found_error(ufunc, dtypes);
 }
 
+
+/*
+ * Support for masked inner-strided loops. These are currently ONLY used
+ * for normal ufuncs, and only a generic loop getter exists.
+ * It may make sense to generalize this in the future or allow specialization.
+ * Until then, the inner-loop signature is flexible.
```

Comment:
Oh, what I meant is: It can make sense to make this a customizable method of `ArrayMethod`, so users can set it for an individual `ArrayMethod`.

That would be clearer (and the comment would make more sense if it was in the `array_method.c` file, but I thought I would move the code separately).

Reply:
I assume @seberg will revisit the comment in a future PR.